test/transaction.js

```diff
@@ -205,6 +205,29 @@ describe('Transaction', function () {
   })
 
   describe('hashForSignature', function () {
+    it('only uses V0 serialization', function () {
+      var randScript = new Buffer('6a', 'hex')
+
+      var tx = new Transaction()
+      tx.addInput(new Buffer('0000000000000000000000000000000000000000000000000000000000000000', 'hex'), 0)
+      tx.addOutput(randScript, 5000000000)
+      tx.___toBuffer = tx.__toBuffer
```

Comment:
Hahaha, no not the transaction.  

I meant,  `tx.__original_toBuffer = tx.__toBuffer` or something, rather than just an extra `_`